src/main/java/io/github/fileanalysissuite/adaptersdk/impls/jaxrs/internal/utils/ItemMetadataFunctions.java

```diff
@@ -42,15 +41,13 @@ public static io.github.fileanalysissuite.adaptersdk.impls.jaxrs.internal.server
                 .name(itemMetadata.getName())
                 .title(itemMetadata.getTitle())
                 .size(itemMetadata.getSize())
-                .createdTime(Optional.ofNullable(itemMetadata.getCreatedTime()).map(Instant::toString).orElse(null))
-                .accessedTime(Optional.ofNullable(itemMetadata.getAccessedTime()).map(Instant::toString).orElse(null))
-                .modifiedTime(itemMetadata.getModifiedTime().toString())
+                .createdTime(Optional.ofNullable(itemMetadata.getCreatedTime())
+                    .map(t -> InstantFunctions.toRFC3339DateTimeString(t)).orElse(null))
```

Comment:
You should generally prefer to use the member syntax rather than a lambda when there is an option.

Reply:
Now using member syntax